packages/console/src/consts/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './logs';
 
 export const themeStorageKey = 'logto:admin_console:theme';
 export const requestTimeout = 20_000;
+export const generatedPasswordStorageKey = 'logto:admin_console:generated_password';
```

Comment:
will this key also be used in reset password use case? if the answer is yes, should we use a more general key name?

Reply:
NO